x/evm/keeper/grpc_query.go

```diff
@@ -481,12 +481,17 @@ func (q Querier) TokenInfo(c context.Context, req *types.TokenInfoRequest) (*typ
 		}
 	}
 
+	burnerCodeHashHex := ""
+	if burnerCodeHash, ok := token.GetBurnerCodeHash(); ok {
+		burnerCodeHashHex = burnerCodeHash.Hex()
```

Comment:
Could also return the latest burner code hash instead, since external tokens always use the latest